src/ORM/Connect/PDOQuery.php

```diff
@@ -22,16 +22,52 @@ class PDOQuery extends Query
      * Hook the result-set given into a Query class, suitable for use by SilverStripe.
      * @param PDOStatement $statement The internal PDOStatement containing the results
      */
-    public function __construct(PDOStatement $statement)
+    public function __construct(PDOStatement $statement, PDOConnector $conn)
```

Comment:
Is `DB::get_conn()->getDriver()` an acceptable alternative to changing this method signature? I don’t feel too strongly about it given this is fairly internal